packages/firebase_auth/ios/Classes/FirebaseAuthPlugin.m

```diff
@@ -239,8 +239,8 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     [[self getAuth:call.arguments].currentUser
         reauthenticateAndRetrieveDataWithCredential:[self getCredential:call.arguments]
                                          completion:^(FIRAuthDataResult *r,
-                                                      NSError *_Nullable error) {
-                                           [self sendResult:result forObject:nil error:error];
+                                                      NSError *error) {
+                                           [self sendResult:result forAuthDataResult:nil error:error];
```

Comment:
i should be `forAuthDataResult:r` with `r` is `FIRAuthDataResult `

Reply:
Oh, I didn't notice that. Thank you!